src/dlr_tvm.cc

```diff
@@ -170,6 +171,7 @@ void TVMModel::GetOutputSizeDim(int index, int64_t* size, int* dim) {
 void TVMModel::Run() {
   tvm::runtime::PackedFunc run = tvm_module_->GetFunction("run");
   run();
+  CallHome(MODEL_RUN, model_path_);
```

Comment:
What will be performance penalty for calling CallHome in Run?

Reply:
We can provide performance stats on x86_64 and armeabi AVD. Please elaborate more on ec2 box for Android.

Reply:
On x86_64 AVD, the performance penalty up to 1.08x. Without the feature latency 14.099 sec (100 inference of GLUONCV_RESNET18_V2 model) and with feature latency 14.752 sec.